src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -248,7 +249,23 @@ public SCMSourceCriteria getSCMSourceCriteria(SCMSource source) {
             criteriaTL.set(old);
         }
     }
-    
+
+    /**
+     * Will create an specialized view when there are no repositories found, which contain a Jenkinsfile
+     */
+    @Override
+    public View getPrimaryView() {
+        if (getItems().isEmpty()) {
+            return new EmptyOrganizationFolderView(this);
+        }
+        return super.getPrimaryView();
+    }
+
+    @Override
+    public AbstractFolderDescriptor getDescriptor() {
+        return new DescriptorImpl();
+    }
```

Comment:
The getDescriptor()? Was returning null otherwise, NPE. Should I do something else to ensure it works properly?